src/components/Academy/Enterprise-Section/EnterpriseSection.js

```diff
@@ -0,0 +1,283 @@
+import React from "react";
+import styled from "styled-components";
+
+const EnterpriseSectionWrapper = styled.section`
+  font-family: "Qanelas Soft", sans-serif;
```

Comment:
Why is this not standard? @vr-varad @ARYANSHAH1567, let’s be on a mission to ensure that we are reusing components and reusing styles across this site without defining new styles. Please make this a topic of discussion for tomorrow’s meeting.